st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -376,7 +376,7 @@ def get_all(self, exclude_attributes=None, **kw):
         if 'timestamp_lt' in kw:
             query_options = {'sort': ['-start_timestamp', 'action.ref']}
             kw['query_options'] = query_options
-        elif 'timestamp_gt' in kw:
+        elif 'timestamp_gt' in kw or 'sort_desc' in kw:
```

Comment:
Actually, shouldn't this be `sort_asc`, since `+start_timestamp` implies sorting in ascending order (smallest aka oldest executions first)?

I know it's probably a bit confusing right now since CLI options and filters don't always map directly to the API ones and we do some additional formatting and sorting in the CLI...